### DISCUSSION TITLE
自定义顶栏的稍后在看和收藏没有显示总视频数量

### DISCUSSION THREAD

**Discussion by wjwkill** [the1812/Bilibili-Evolved#4069] [功能建议 / Ideas]
有一个建议，自定义顶栏的稍后在看和收藏没有显示总视频数量，原版是有的，希望能够加上！
![image](https://user-images.githubusercontent.com/49703448/223337208-cd0796d7-5bcf-4a20-b4c4-9d9c8dcb7963.png)
![image](https://user-images.githubusercontent.com/49703448/223337273-56b90a79-6b91-40dc-ab8d-7998e3e6eb62.png)
![image](https://user-images.githubusercontent.com/49703448/223337303-4182be21-e5f2-4356-8f90-ab74152dbb2d.png)

**Comment by Geeyun-JY3** [comment]
~重复：#1516~
要不还是合并到这里好了，#1516 也没提到稍后再看🌚

**Reply by the1812** [reply]
#1516 那时候还是 v1 做不了, 现在可以安排下了
